mod/groups/start.php

```diff
@@ -81,7 +81,10 @@ function groups_init() {
 
 	//extend some views
 	elgg_extend_view('elgg.css', 'groups/css');
-	elgg_extend_view('elgg.js', 'groups/js');
+	if (_elgg_view_may_be_altered('groups/js', __DIR__ . '/views/default/groups/js.php')) {
```

Comment:
@mrclay I am confused about `_elgg_view_maybe_altered()`. How does this tell us if the plugin is overwriting the view?

```php
	// check location
	$root = dirname(dirname(__DIR__));
	$expected_path = "$root/views/$viewtype/" . ltrim($path_from_viewtype, '/\\');
```